### PR TITLE
OGM-1495 Generate @SequenceGenerator source id values with cluster counter

### PR DESCRIPTION
**Pull request by fax4ever**
https://hibernate.atlassian.net/browse/OGM-1495

### CODE REVIEW

File: infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml [hunk 1]
```diff
@@ -99,6 +99,7 @@
         <subsystem xmlns="urn:infinispan:server:core:9.2">
             <cache-container name="clustered" default-cache="default" statistics="true">
                 <transport lock-timeout="60000"/>
+                <global-state/>
```
Comment (DavideD) [inline infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml]: Remember that it's nice to have in the commit also a "why" you are doing something ( not necessarly in the first line of the commit)
In this case, why do you need to add it back?

Comment (fax4ever) [reply infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml]: File *wildfly-trimmed-config.xml* is a subset of cluster.xml configuration. The global state is present in all Infinispan server provided configurations (standalone.xml, clustered.xml, domain profiles) and it is necessary to allow to the server to store persistent counters data.

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-ogm/pull/1041

I've added this to one of the commit before merging it:

> File wildfly-trimmed-config.xml is a subset of cluster.xml configuration. The global state is present in all Infinispan server provided configurations (standalone.xml, clustered.xml, domain profiles) and it is necessary to allow to the server to store persistent counters data.

Thanks

**Comment by fax4ever** [comment]
Thank you!

**Pull request closed without merging**